lib/qesapdeployment.pm

```diff
@@ -242,7 +244,7 @@ sub qesap_venv_cmd_exec {
     $cmd .= 'set -o pipefail ; ' if $args{log_file};
     $cmd .= join(' ', 'timeout', $args{timeout}, $args{cmd});
     # always use tee in append mode
-    $cmd .= "|& tee -a $args{log_file}" if $args{log_file};
+    $cmd .= " |& tee -a $args{log_file}" if $args{log_file};
```

Comment:
If missing, this space is enough to break the `--junit` glue script feature.

Reply:
Never wrong to prepend the string with a blank space when doing `$cmd .=` .... you never now what was the last character in `$cmd` and the shell should be able to deal with extra spaces flawlessly.

Reply:
And in this case we know that it didn't end with a space.

Reply:
And in this case we know from the line right before that it doesn't end in a blank space :)

Reply:
Issue is now fixed but I leave a note for myself to check what is going on within the glue script when doing `qesap.py ... ansible --junit /tmp/results/|& tee something.txt` without the space. Here seems not an issue http://openqaworker15.qa.suse.cz/tests/297728/logfile?filename=serial_terminal.txt but it is in other executions.